UPGRADING.md

```diff
@@ -218,7 +214,17 @@ From here on the doc will be focussed on an ODFE plugin as an example.
 
 See [anomaly-detection#35](https://github.com/opensearch-project/anomaly-detection/pull/35) for an example.
 
-### OpenSearch Dashboard Plugins
+##### Indices
+
+You can adopt one of the following strategies.
+
+1. Don't change index names. For example, `.opendistro-anomaly-detectors` remains as such.
```

Comment:
I think we need to understand this from a product perspective. 
From what I understand, we do not want to have `opendistro` in the future. 
Probably @elfisher can help us answer this.

So should we offer it as a strategy? i feel we should not as sooner or later we might have to change and it would be the same effort unless the product team is ok having `opendistro` forever.

Reply:
Agreed. Let's discuss this before merging.